workers/unity/Packages/com.improbable.gdk.core/Worker/Worker.cs

```diff
@@ -170,12 +167,18 @@ private static async Task<DeploymentList> GetDeploymentList(Locator locator)
             using (var deploymentsFuture = locator.GetDeploymentListAsync())
             {
                 var deploymentList = await Task.Run(() => deploymentsFuture.Get()).ConfigureAwait(false);
-                if (!string.IsNullOrEmpty(deploymentList.Error))
+                // Guard against null refs. This shouldn't be triggered.
+                if (!deploymentList.HasValue)
```

Comment:
Had a discussion with Worker about this. The reason it returns a nullable is that `Future<T>` expects a reference type, so the `DeploymentList` should always be set, but the `Error`/`List<Deployment>` inside may not be.

This is just defensive coding such that if any such regression occurred it would be immediately obvious.

Reply:
which would bypass this anyway and propagate to the function awaiting this.